pom.xml

```diff
@@ -403,30 +408,9 @@
                     </configuration>
                 </plugin>
                 <plugin>
-                    <groupId>org.codehaus.mojo</groupId>
-                    <artifactId>codenarc-maven-plugin</artifactId>
```

Comment:
What's wrong with CodeNarc? It doesn't work with 8?

Reply:
it doesn't work with JDK 11, I'll keep it with JDK8.

it uses by default codenarc 0.22 and groovy-all 1.7.something.
Groovy 1.7.x isn't compatible with JDK11.
Latest versions of codenarc is 1.2.1 and of groovy-all is 2.5.3.
Both is configurable with codeNarcVersion and groovyVersion, but groovy-all download fails because it's pom type in this version.

codenarc-maven-plugin development doesn't seem to active since 2015